docs/custom_skip_members.py

```
@@ -31,0 +1,15 @@
+from include_exclude import skip_dict, keep_dict
+
+
+def autoapi_skip_members(app, what, name, obj, skip, options):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)